eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/DefaultEventCriteria.java

```diff
@@ -1,58 +0,0 @@
-/*
- * Copyright (c) 2010-2025. Axon Framework
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.axonframework.eventsourcing.eventstore;
-
-import jakarta.annotation.Nonnull;
-
-import java.util.Set;
-
-import static java.util.Objects.requireNonNull;
-
-/**
- * Implementation of the {@link EventCriteria} that takes both message types and their tags into consideration.
- *
- * @param types The types that this criteria instance matches with.
- * @param tags  The tags that events are expected to have.
- * @author Steven van Beelen
- * @author Allard Buijze
- * @since 5.0.0
- */
-record DefaultEventCriteria(@Nonnull Set<String> types,
```

Comment:
This was morphed into `FilteredEventCriteria`, but apparently the changes are too big to see it as a file move.